### PR TITLE
nttw 3 updates

### PR DESCRIPTION
**Pull request by dericed**
ping @ablwr @JeromeMartinez

### CODE REVIEW

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 1]
```diff
@@ -1,40 +1,58 @@
 {% extends 'NoTimeToWaitBundle::base.html.twig' %}
 
-{% block title %}No Time to Wait 3: {% endblock %}
+{% block title %}No Time to Wait 3{% endblock %}
 
 {% block body %}
-<h1><strong>No Time to Wait!</strong> *</h1>
+<h1>* <strong>No Time to Wait!</strong> *</h1>
```
Comment (ablwr) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: Asterisks for ...decoration?

Comment (JeromeMartinez) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: the asterisk on the right was taken from nttw1 & 2, the meaning is at the bottom. No change compared to last year ;-).
So the question is more about why the asterisk on the left, of if we should remove the asterisk on the right.

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 2]
```diff
@@ -1,40 +1,58 @@
...
+<p>Thursday, October 25, 2018 and Friday, October 26, 2018</p>
 
+<p>Opening: 09:00<br />
+Closing: 17:00</p>
 
 <h2>Where</h2>
 <p><a href="http://www.bfi.org.uk/">British Film Institute</a>, Belvedere Rd, South Bank, London SE1 8XT, UK</p>
 
-<p>Parts of the workshop will also be accessible remotely</p>
+<p><a href="https://www.filmmuseum.at">Österreichisches Filmmuseum - Austrian Film Museum</a></p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: copy/paste mistake?

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 3]
```diff
@@ -1,40 +1,58 @@
 {% extends 'NoTimeToWaitBundle::base.html.twig' %}
 
-{% block title %}No Time to Wait 3: {% endblock %}
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig:3]: NTTW 1 & 2 had a title after the ":". No title for NTTW3?

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]
At least copy/paste mistakes to fix.
And maybe clarifications (asterisk and "missing" title)

**Comment by dericed** [comment]
ping to @JeromeMartinez, @ablwr. This is ready for a new review.

**Comment by JeromeMartinez** [comment]
You still have 2 (similar) images in the PR and you don't use it anymore. you can drop them.

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 4]
```diff
@@ -1,63 +1,83 @@
...
 
+<h2>Sponsorship</h2>
+<p>The organizers of No Time to Wait strive to maintain the symposium as a free registration event through efficient and transparent budgeting and via the generosity of our hosts and sponsors. The sponsors of No Time to Wait help make the discussions, presentations, and momentum of the No Time to Wait event possible and enable the event to maintain a high level of accessibility and involvement. Sponsors shall be acknowledged via the symposium's website, printed programs, and presence at the sympoisium. Sponsors may indicate that their contribution is allocated to a specific need, such as funding travel stipends, hosting a reception, or supporting live streaming. To discuss or inquire regarding sponsorship opportunties please <a href="{{ path('ma_contact') }}">please contact us</a>.
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: typo "sympoisium"

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig [hunk 5]
```diff
@@ -1,63 +1,83 @@
...
 <p>(To Be Announced)</p>
 
 <p>If you would like to sponsor additional travel grants, <a href="{{ path('ma_contact') }}">please contact us</a>.</p>
 
-
 <h2>Program:</h2>
 
-<p>(To Be Announced)</p>
-
+<p>(To Be Announced) A Call for Proposals shall be issued soon.</p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: isn't Call for Proposals URL already available?

Comment (ablwr) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw3.html.twig]: Announce first, update with active CFP later?

**Comment by dericed** [comment]
Comments resolved, ready for re-review.

**Review by ablwr — APPROVED** [review]

**Review by JeromeMartinez — APPROVED** [review]

**Pull request merged**